source/end-user/integration/terraform.rst

```diff
@@ -1,3 +1,156 @@
 =========
 Terraform
 =========
+
+OpenStack Provider
+==================
+
+* https://www.terraform.io/docs/providers/openstack/index.html
+
+* Create a ``modules.tf`` file
+
+.. code-block:: json
+
+   provider "openstack" {
+     version = "~> 1.6"
+   }
+
+   terraform {
+     required_version = "= 0.11.7"
```

Comment:
Why do we have to pin the required version?

Reply:
No we need not pin the required version.